submissions/bilkisuTace/html-forms/contactUs.html

```diff
@@ -0,0 +1,103 @@
+	<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Contact Us | Ladies & Gents</title>
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet"/>
+	<link rel="stylesheet" type="text/css" href="../css/contactus.css"/>
+</head>
+<body>
+	<div class="container">
+		<div class="topnav">
+			<div id="nav-logo" class="nav-logo">
+				<a href="#">
+                    <img src="../html/images/logo.jpg"/>
+                </a>
+			</div>
+			<div class="nav-links">
+				<ul>
+                    <li><a href="../html/product.html">Product Page</a></li>
+                    <li><a href="contactUs.html">Contact Us</a></li>
+                    <li><a href="signup.html">Sign-up</a></li>
+                </ul>
+			</div>
+		</div>
+		<div class="page-body">
+			<div class="body-text">
+				<span>
+					<img class="img" src="../html/images/call.png" alt="">
+					<p class="p-heading"><b>Please contact us via these numbers:</b></p>
+					<p>+23480808001</p>
+					<p>+23480808002</p><br/>
+					<img class="img" src="../html/images/env1.png" alt="">
+					<p class="p-heading"><b>Email Address:</b></p>
+					<p>support@ladies&gents.com</p><br>
+					<p class="p-heading"><b>Or Connect with us:</b></p>
+					<ul class="body-social">
+                        <li><img src="../html/images/facebook.jpg"/></li>
+                        <li><img src="../html/images/twitter.jpg"/></li>
+                        <li><img src="../html/images/gmail.jpg"/></li>
+                        <li><img src="../html/images/youtube.jpg"/></li>
+                    </ul>
+				</span>
+			</div>
+			<div class="div-form">
+				<span class="body-form">
+					<form>
+						<fieldset>
+						<img src="../html/images/env2.jpg" alt="">
+						<p class="p-heading"><b>Or better still send us a message</b></p>
+						<input class="input" type="text" name="fullname" id="fullname" placeholder="Full Name"><br/><br/>
+						<input class="input" type="email" name="email" id="email" placeholder="Email Address Here"><br/><br/>
+						<input class="input" type="text" name="subject" id="subject" placeholder="Subject"><br/><br/>
+						<label class="message" for="textarea">Message</label><br>
+						<textarea class="text-area"></textarea><br/><br/>
+						<button class="button" type="submit">Send</button>
+						</fieldset>
+					</form>
+				</span>
+			</div>
+		</div>
+		<div class="footer">
+			<div class="footer-body">
+				<div class="footer-links">
+					<span class="links">
+						<a href="contact.html">Contact Us </a><br/><br/>
```

Comment:
You should put the links in separate span elements or you use list to render them. @bilkisutace Also, remove the br around the links